meeshkan/serve/mock/callbacks.py

```diff
@@ -89,12 +89,12 @@ def _map_results(self, response, result, format, response_type):
 
         return response
 
-    def __call__(self, request, response):
+    def __call__(self, request, response, storage):
```

Comment:
Should we set the default `storage=None` case for backward compatibility and slightly more flexible usage?

Reply:
It is for `callback_manager' call and it doesn't affect backward compatibility because it's not used outside the core meeshkan stuff. What affects backward compatibility is changes in the Storage interface. It is a container of entities rather than a dict. And it is rather bad. Probably, we could alias access to `storage.default` as just `storage['some_value'].
It will make it less clear but preserve the old behaviour.